tests/models/test_default_evaluator.py

```diff
@@ -1578,3 +1579,30 @@ def test_evaluation_metric_name_configs(prefix, log_metrics_with_dataset_info):
 
     # Dataset info should only be included in logged metric names
     assert all("on_data_iris" not in metric_name for metric_name in result.metrics)
+
+
+@pytest.mark.parametrize("pos_label", [0, 1])
+def test_evaluation_pos_label(pos_label):
```

Comment:
Can we add parametrized test for "average" as well?

Reply:
Got it, I'll add them.

Reply:
Added